spec/first_and_only_spec.rb

```diff
@@ -8,8 +8,10 @@
 
   shared_examples_for "it_is_empty" do
     specify do
-      expect { subject.first_and_only! }.to \
-        raise_error(Enumerable::FirstAndOnly::LengthNotOne, "0")
+      expect { subject.first_and_only! }.to raise_error(
+        Enumerable::FirstAndOnly::CountNotOne,
+        "Expected the count to be 1, but it was 0."
```

Comment:
Maybe I'm doing it wrong, but the original intent was that the error class itself would be part of the error message:

```
Enumerable::FirstAndOnly::CountNotOne # "Oh, I see the count is supposed to be 1"
```

So putting that in the text of the message is kind of redundant:

```
Expected the count to be 1, but it was 0.
```

So, I should mark this as "wont-fix, change the downstream problem with `ActionView`" :trollface: 

So thinking about this in real time,  I think I agree with your change, even though it duplicates information in the class name.  For example, maybe the text of the error message could be localized, but of course, the `CountNotOne` class name would not be localized.

Thanks @mikegee!